examples/font.ml

```diff
@@ -0,0 +1,9 @@
+
+open Graphics
+
+let () =
+  open_graph "";
+  set_font "Liberation Mono";
```

Comment:
It's nice to have a test, thanks.  It might be possible to make the test a bit more generic.  I'll suggest something later.